allensdk/test/brain_observatory/behavior/behavior_project_cache/conftest.py

```diff
@@ -161,6 +166,160 @@ def s3_cloud_cache_data():
     return all_versions, versions
 
 
+@pytest.fixture
+def vbn_s3_cloud_cache_data():
```

Comment:
Can you please rename the existing `s3_cloud_cache_data` to `vbo_s3_cloud_cache_data`? I know that's going to force you to change a lot of boilerplate test instantiations, but it will be good for us in the future to know the difference between these two fixtures.

Reply:
renamed